plugins/jvm-abi-gen/testData/compare/privateCompanionClasses/base/test.kt

```diff
@@ -0,0 +1,5 @@
+package test
+
+class Class {
+    private companion object
```

Comment:
Instead of this separate test, please declare a companion object in `nestedPrivateClasses`.